src/upload.rs

```diff
@@ -393,11 +420,9 @@ fn http_agent() -> Result<ureq::Agent, UploadError> {
 
     let mut builder = ureq::builder().try_proxy_from_env(true);
     let mut tls_builder = native_tls::TlsConnector::builder();
-    if let Some(ca_bundle) = tls_ca_bundle() {
-        let mut reader = io::BufReader::new(File::open(ca_bundle)?);
-        for cert in rustls_pemfile::certs(&mut reader) {
-            let cert = cert?;
-            tls_builder.add_root_certificate(native_tls::Certificate::from_pem(&cert)?);
+    if let Some(root_certs) = ca_certificates()? {
+        for cert in root_certs {
+            tls_builder.add_root_certificate(native_tls::Certificate::from_der(&cert)?);
         }
     }
```

Comment:
I think we should really be calling `tls_builder.disable_built_in_roots(true)` in this block. That would match the behavior on the `rustls` side where we replace the root certs instead of appending to it.

Reply:
I guess we can do it in v2.0 to avoid potential breakage.